changes/v5.0.20-en.md

```diff
@@ -0,0 +1,5 @@
+# v5.0.20
+
+## Bug Fixes
+
+- [#10098](https://github.com/emqx/emqx/pull/10098) When configuring mongodb authorization the mongodb connector crashed with an error in the log file. The reason was that the collection name had the wrong type.
```

Comment:
```suggestion
- Fix mongodb authorization crash when `collection` is configured.
```

Reply:
The crash does not happen when the collection is configured. It happens when the authorization system tries to query the mongodb database.